src/lib/operators/insert.cpp

```diff
@@ -18,7 +18,7 @@ namespace opossum {
 // We need these classes to perform the dynamic cast into a templated ValueColumn
 class AbstractTypedColumnProcessor {
  public:
-  virtual ~AbstractTypedColumnProcessor() = default;
+  virtual ~AbstractTypedColumnProcessor() {}
```

Comment:
why does it prefer this over `=default`?

Reply:
It doesn't. My fault.